pkg/batcher/message_batcher.go

```diff
@@ -21,9 +21,9 @@ import "time"
 type Direction = byte
 
 const (
-	UNKNOWN Direction = iota
-	IN
-	OUT
+	Unknown Direction = iota
+	In
+	Out
```

Comment:
suggestion: since Golang does not have enums, a good practice is to add a suffix indicating the type, e.g. `UnknownDirection`